front/lib/tracking/amplitude/client/generated/index.ts

```diff
@@ -155,6 +155,10 @@ export class AssistantBuilderStepViewed implements BaseEvent {
   }
 }
 
+export class HelpDrawerOpened implements BaseEvent {
```

Comment:
Was that added manually or was that generated by pulling the Amplitude types using the CLI?